publish/publish.proj

```diff
@@ -162,7 +162,7 @@
 	       copying them to the indexed directory -->
       <PublishDirectory Condition="'$(WindowsSdkDir)' == ''">$(DownloadDirectory)</PublishDirectory>
       <!-- list of packages that we want to embed symbol signatures in -->
-      <SymbolPackagesToPublishGlob Condition="'$(SymbolPackagesToPublishGlob)' == ''">$(DownloadDirectory)/**/*.nupkg</SymbolPackagesToPublishGlob>
+      <SymbolPackagesToPublishGlob Condition="'$(SymbolPackagesToPublishGlob)' == ''">$(DownloadDirectory)/**/*.symbols.nupkg</SymbolPackagesToPublishGlob>
```

Comment:
Sorry, missed this comment. Currently, it has to be a property because BuildTools expects it to be to run some of its targets. I don't think I properly anticipated how Core-Setup was going to use this functionality.